### PR TITLE
92 bug report unitfulmatrixx where x is not a vector causes matrixnothing error

### PR DESCRIPTION
**Pull request by ggebbie**
Edit PR #93 (did not succeed in updating previous PR and added this one which supercedes #93 )

### CODE REVIEW

**Comment by ggebbie** [comment]
@singularitti please check this branch to see if it solves the issue with StaticArrays for you.

**Comment by singularitti** [comment]
It seems to be fixed for me. Thank you!

File: src/UnitfulMatrix.jl [hunk 1]
```diff
@@ -164,9 +166,13 @@ end
 function UnitfulMatrix(a::AbstractVector) # should be called UnitfulVector?
     numbers = ustrip.(a)
     M = size(numbers)
-    unitrange = Vector{Unitful.Units}(undef,M)
+    unitrange = Vector{Unitful.FreeUnits}(undef,M)
 
-    unitrange = unit.(a)
+    #println(typeof(vec(unit.(a))))
```
Comment (singularitti) [inline src/UnitfulMatrix.jl:171]: What about removing these comments?

Comment (ggebbie) [reply src/UnitfulMatrix.jl]: thank you --- done

**Pull request merged**